src/settings.py

```
@@ -0,0 +1,14 @@
+from pydantic_settings import BaseSettings
+
+
+class Settings(BaseSettings):
+
+    PROJECT_NAME: str = 'py-fastapi-city-temperature-management-api'
+
+    SQLITE_DATABASE_URL: str | None = "sqlite+aiosqlite:///./city_temperature.db"
```

`SQLITE_DATABASE_URL` is typed `str | None` but given a default. If you intend the URL to be required in some deployments, validate or assert its presence where used (e.g., raise a clear error if `None`) to fail fast instead of producing confusing engine errors later.